public/zxw_factory_config.xml

```diff
@@ -259,11 +259,11 @@
     <SupportUIList>
         <Item id="45" ui="KSW_ALFA_ROMEO" />
 
-        <Item id="23" ui="KSW_AUDI" />
-        <Item id="24" ui="KSW_AUDI_FY" />
-        <Item id="27" ui="KSW_AUDI_TY" />
+        <Item id="23" ui="KSW_AUDI_MIB3" />
+        <Item id="24" ui="KSW_AUDI_MIB3_FY" />
+        <Item id="27" ui="KSW_AUDI_MIB3_TY" />
         <!-- <Item id="29" ui="KSW_AUDI_CADILLAC" /> Invalid ID -->
-        <Item id="38" ui="KSW_GT6_AUDI" />
+        <Item id="38" ui="KSW_AUDI_MMI_4G" />
```

Comment:
Cool and I was just gonna hit approve since I'd like that consistency, but…

ZXW firmwares have these exact names in the source code so it's easier for me to track what's new:

<img width="1512" alt="Screenshot 2024-10-28 at 15 42 12" src="https://github.com/user-attachments/assets/c2a5f741-b36c-4896-be40-c98d0ec4e186">

For example the following piece:

```java
public static final int UI_NUM_COMMON_GS = 26;
public static final int UI_NUM_KSW_ALFA_ROMEO = 45;
public static final int UI_NUM_KSW_AUDI = 23;
public static final int UI_NUM_KSW_AUDI_BENTLEY = 40;
public static final int UI_NUM_KSW_AUDI_CADILLAC = 29;
public static final int UI_NUM_KSW_AUDI_FY = 24;
public static final int UI_NUM_KSW_AUDI_PORSCHE = 43;
public static final int UI_NUM_KSW_AUDI_TY = 27;
public static final int UI_NUM_KSW_BENZ = 18;
public static final int UI_NUM_KSW_BENZ_FY = 22;
public static final int UI_NUM_KSW_BENZ_FY_WITHOUT_WEATHER = 46;
public static final int UI_NUM_KSW_BENZ_LS = 48;
public static final int UI_NUM_KSW_BENZ_NTG6_0_2021 = 21;
public static final int UI_NUM_KSW_BENZ_NTG6_0_2021_V1 = 31;
public static final int UI_NUM_KSW_BENZ_V1 = 30;
public static final int UI_NUM_KSW_BMW_ID6 = 42;
public static final int UI_NUM_KSW_BMW_ID7 = 16;
public static final int UI_NUM_KSW_BMW_ID8 = 17;
public static final int UI_NUM_KSW_BMW_ID8_BFD = 49;
public static final int UI_NUM_KSW_BMW_ID8_COPILOT = 41;
public static final int UI_NUM_KSW_BMW_ID8_KJD = 47;
public static final int UI_NUM_KSW_COMMON_ID7 = 44;
public static final int UI_NUM_KSW_Common_UI_GS_UG = 34;
public static final int UI_NUM_KSW_Common_UI_GS_UG_1024 = 35;
public static final int UI_NUM_KSW_GT6_AUDI = 38;
public static final int UI_NUM_KSW_LANDROVER = 25;
public static final int UI_NUM_KSW_LANDROVER_BLUE_WHITE = 32;
public static final int UI_NUM_KSW_LANDROVER_COPILOT = 39;
public static final int UI_NUM_KSW_MBUX_1024 = 33;
public static final int UI_NUM_KSW_PEMP_BMW_ID7 = 19;
public static final int UI_NUM_KSW_PEMP_BMW_ID8 = 20;
public static final int UI_NUM_KSW_SAILOR_T001 = 100;
public static final int UI_NUM_KSW_SAILOR_T002 = 101;
public static final int UI_NUM_KSW_SAILOR_T003 = 102;
public static final int UI_NUM_LEXUS_LS_UI = 37;
public static final int UI_NUM_LEXUS_UI = 36;
```

So for GT6/GT7 firmwares I just remove the `UI_NUM_` prefix and stick these labels in this `zxw_factory_config.xml` file and in the themes section, both in the folder structure and markdown files, like this https://github.com/kamilbrk/headunits/blob/main/src/data/themes/zxw/49-KSW_BMW_ID8_BFD/index.md?plain=1

So I'm kinda second guessing this PR as it stands, any opinion?

Reply:
I think I'd likely do the same in your position, and as long as the UI names in the config match the values on the themes page, I think we're fine. I'll just edit them locally for my own use :)